Lesson33/Selcan/lesson33-app/src/Exercise10/index.jsx

```diff
@@ -0,0 +1,24 @@
+// Exercise 10: Double Click Event
+// Implement an event handler that triggers when a button is double-clicked. Display an alert indicating that the button was double-clicked.
+    
+import { useState } from "react";
+
+export const Exercise10 = () => {
+  const [items, setItems] = useState(['Button1', 'Button2', 'Button3']);
+
+  function clickHandler(item) {
+    alert(`You doubleClicked ${item}`);
+  }
+
+  return (
+    <ul>
+      {items.map((item, index) => (
+        <li key={index}>
```

Comment:
In React, you need to use unique identifier for this key property. However, index of element is not a stabile identifier. In this case, it would be better to use `item` value as the key.

```suggestion
        <li key={item}>
```